velox/functions/remote/server/RemoteFunctionService.h

```diff
@@ -47,9 +53,9 @@ class RemoteFunctionServiceHandler
       exec::EvalErrors* evalErrors,
       const std::unique_ptr<VectorSerde>& serde) const;
 
+  const std::string functionPrefix_;
   std::shared_ptr<memory::MemoryPool> pool_{
       memory::memoryManager()->addLeafPool()};
```

Comment:
we can probably remove this from here now

Reply:
nice catch. I was surprised it didn't crash my test since it doesn't init the memory manager. looks like the compiler erases / elides this expression.